lonestar/mining/cpu/motif-counting/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+app_mining_cpu(motif motif-counting)
+
+add_test_mine(small1 motif-counting -symmetricGraph -simpleGraph "${BASEINPUT}/Mining/citeseer.csgr")
```

Comment:
Same as above. Just leave this commented out to avoid depending on https://github.com/IntelligentSoftwareSystems/Galois/issues/143.